actor/src/main/scala/org/apache/pekko/dispatch/VirtualThreadSupport.scala

```diff
@@ -93,8 +93,8 @@ private[dispatch] object VirtualThreadSupport {
       }
       val nameMethod = ofVirtualClass.getDeclaredMethod("name", classOf[String], classOf[Long])
       val factoryMethod = builderClass.getDeclaredMethod("factory")
-      val zero = java.lang.Long.valueOf(0L)
```

Comment:
How about declaring a constant in an `object VirtualThreadSupport` with `val zero = java.lang.Long.valueOf(0L)`? Then you avoid boxing in the new code. 0L has to be boxed to a java.util.Long on every call.

Reply:
@pjfanning https://github.com/apache/pekko/issues/2170

I think we should just drop the counter, wdyt?